plugin.video.mytv_bg/resources/language/Bulgarian/strings.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
When you have some time please consider to move the strings to the new .po format. You don't need to do it for now though. See : http://kodi.wiki/view/language_support